### PR TITLE
Added Njakatiana

### PR DESCRIPTION
**Pull request by Tsikitsiky**
Hey, 
This is my last artist, please have a look.

Thank you so much,
Sarah

### CODE REVIEW

File: n/njakatiana.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Njakatiana
+
+![Njakatiana](njakatiana.jpg)
+
+Njakatiana in his real name Raharisoa Njakatiana is a Malagasy singer, who was born on 07th june 1968. He started his career as a singer in 1992. He is one of the best Malagasy singer.
```
Comment (ndorfin) [inline n/njakatiana.md]: "Njakatiana (his real name is Raharisoa Njakatiana) is…"

File: n/njakatiana.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+**Genre:** [Salegy](https://en.wikipedia.org/wiki/Salegy), Slow
```
Comment (ndorfin) [inline n/njakatiana.md]: No need to mention the link to Salegy here.

**Review by ndorfin — CHANGES_REQUESTED** [review]

File: n/njakatiana.md (hunk 2 above)
Comment (Tsikitsiky) [reply n/njakatiana.md]: I made the changes, please check.

**Review by ndorfin — APPROVED** [review]
Yay, congrats!

**Pull request merged**